app/src/main/kotlin/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -1662,14 +1663,19 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
         if (presenter.isAlwaysShowFirstViewOnAppStartEnabled() &&
             LifecycleHandler.isAppInBackground()
         ) {
-            // Pattern matches urls which are NOT allowed to show the first view after app is started
-            // This is
-            // /config/* as these are the settings of HA but NOT /config/dashboard. This is just the overview of the HA settings
-            // /hassio/* as these are the addons section of HA settings.
-            if (webView.url?.matches(".*://.*/(config/(?!\\bdashboard\\b)|hassio)/*.*".toRegex()) == false) {
-                Timber.d("Show first view of default dashboard.")
-                webView.evaluateJavascript(
-                    """
+            if (serverManager.getServer(presenter.getActiveServer())?.version?.isAtLeast(2025, 6, 0) == true) {
+                sendExternalBusMessage(
+                    NavigateTo("/config", true)
```

Comment:
Replace is only replacing the head of the history. We could combine this with something like 
```
webView.goBackOrForward(0)
```
Then invoke `NavigateTo` to clear the back history.

Reply:
Looking at the [reference](https://developer.android.com/reference/android/webkit/WebView#goBackOrForward(int)), the function you suggests does nothing as the history isn't changed? I'm confused.

Reply:
I didn't look at the impl. I though it was 0 for the first item in the history. Then we could use the size of the history maybe. It was just a suggestion that we could take a look. 

I imagine that someone that never close the app nor restart the device might have an history that can grow pretty big if he navigates within the app a lot. 

We should not tackle this here IMO, we could open an issue to see if people complains about it.

Reply:
But I just saw `webView.clearHistory()` we could use this instead it is meant for this.

Reply:
> we could open an issue to see if people complains about it

Because of how long this feature has existed and the lack of comments about it, it looks to me like most people that use this don't consider it a problem/complaint. An issue to track it wouldn't add a lot of value in my opinion.